go/libraries/doltcore/sqle/dprocedures/dolt_pull.go

```diff
@@ -215,3 +219,22 @@ func stopProgFuncs(cancel context.CancelFunc, wg *sync.WaitGroup, statsCh chan p
 	close(statsCh)
 	wg.Wait()
 }
+
+func checkForUncommittedChanges(ctx *sql.Context, root *doltdb.RootValue, headRoot *doltdb.RootValue) error {
+	rh, err := root.HashOf()
+
+	if err != nil {
+		return err
+	}
+
+	hrh, err := headRoot.HashOf()
+
+	if err != nil {
+		return err
+	}
+
+	if rh != hrh {
```

Comment:
It's not 1968, there is no shortage of storage. Longer names please. How about rootHash and headHash?